2014/06/29 12:29:38 [error] [exception.CException] exception 'CException' with message 'Property "CUrlManager.showScriptFile" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\yii\framework\YiiBase.php(220): CComponent->__set('showScriptFile', false)
#1 C:\xampp\htdocs\yii\framework\base\CModule.php(386): YiiBase::createComponent(Array)
#2 C:\xampp\htdocs\yii\framework\base\CApplication.php(522): CModule->getComponent('urlManager')
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(140): CApplication->getUrlManager()
#4 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#5 C:\xampp\htdocs\yii\framework\purchaseorder\index.php(13): CApplication->run()
#6 {main}
REQUEST_URI=/yii/framework/purchaseorder/gii
---
2014/06/29 12:29:49 [error] [exception.CException] exception 'CException' with message 'Property "CUrlManager.showScriptsFile" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\yii\framework\YiiBase.php(220): CComponent->__set('showScriptsFile', false)
#1 C:\xampp\htdocs\yii\framework\base\CModule.php(386): YiiBase::createComponent(Array)
#2 C:\xampp\htdocs\yii\framework\base\CApplication.php(522): CModule->getComponent('urlManager')
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(140): CApplication->getUrlManager()
#4 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#5 C:\xampp\htdocs\yii\framework\purchaseorder\index.php(13): CApplication->run()
#6 {main}
REQUEST_URI=/yii/framework/purchaseorder/gii
---
2014/06/29 12:30:00 [error] [exception.CException] exception 'CException' with message 'Property "CUrlManager.showScriptFiles" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\yii\framework\YiiBase.php(220): CComponent->__set('showScriptFiles', false)
#1 C:\xampp\htdocs\yii\framework\base\CModule.php(386): YiiBase::createComponent(Array)
#2 C:\xampp\htdocs\yii\framework\base\CApplication.php(522): CModule->getComponent('urlManager')
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(140): CApplication->getUrlManager()
#4 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#5 C:\xampp\htdocs\yii\framework\purchaseorder\index.php(13): CApplication->run()
#6 {main}
REQUEST_URI=/yii/framework/purchaseorder/gii
---
2014/06/29 12:33:02 [error] [php] Missing argument 1 for CModule::getModule(), called in C:\xampp\htdocs\yii\framework\base\CComponent.php on line 111 and defined (C:\xampp\htdocs\yii\framework\base\CModule.php:269)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\purchaseorder\protected\controllers\SiteController.php(33): CWebApplication->__get()
#1 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#5 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#7 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#8 C:\xampp\htdocs\yii\framework\purchaseorder\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/framework/purchaseorder/
2014/06/29 12:33:39 [error] [php] Missing argument 1 for CModule::getModule(), called in C:\xampp\htdocs\yii\framework\base\CComponent.php on line 111 and defined (C:\xampp\htdocs\yii\framework\base\CModule.php:269)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\base\CComponent.php(264): CWebApplication->__get()
#1 C:\xampp\htdocs\yii\framework\purchaseorder\protected\controllers\SiteController.php(33): CWebApplication->__call()
#2 C:\xampp\htdocs\yii\framework\purchaseorder\protected\controllers\SiteController.php(33): CWebApplication->module()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii\framework\purchaseorder\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/framework/purchaseorder/
2014/06/29 12:33:39 [error] [php] Missing argument 1 for CModule::getModule(), called in C:\xampp\htdocs\yii\framework\base\CComponent.php on line 111 and defined (C:\xampp\htdocs\yii\framework\base\CModule.php:269)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\base\CComponent.php(264): CWebApplication->__get()
#1 C:\xampp\htdocs\yii\framework\purchaseorder\protected\controllers\SiteController.php(33): CWebApplication->__call()
#2 C:\xampp\htdocs\yii\framework\purchaseorder\protected\controllers\SiteController.php(33): CWebApplication->module()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii\framework\purchaseorder\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/framework/purchaseorder/
2014/06/29 12:40:05 [error] [php] Trying to get property of non-object (C:\xampp\htdocs\yii\framework\purchaseorder\protected\controllers\SiteController.php:33)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#5 C:\xampp\htdocs\yii\framework\purchaseorder\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/framework/purchaseorder/
2014/06/29 12:51:37 [error] [system.db.CDbCommand] Error in preparing SQL: CREATE TABLE '{{user_remote_identities}}' (
	"id" integer PRIMARY KEY AUTOINCREMENT NOT NULL,
	"user_id" integer NOT NULL REFERENCES {{users}} (id) ON UPDATE CASCADE ON DELETE CASCADE,
	"provider" varchar(100) NOT NULL,
	"identifier" varchar(100) NOT NULL,
	"created_on" timestamp NOT NULL,
	"last_used_on" timestamp
)
2014/06/29 12:51:37 [error] [system.db.CDbCommand] CDbCommand::execute() failed: CDbCommand failed to prepare the SQL statement: SQLSTATE[HY000]: General error: 1 unrecognized token: "{". The SQL statement executed was: CREATE TABLE '{{user_remote_identities}}' (
	"id" integer PRIMARY KEY AUTOINCREMENT NOT NULL,
	"user_id" integer NOT NULL REFERENCES {{users}} (id) ON UPDATE CASCADE ON DELETE CASCADE,
	"provider" varchar(100) NOT NULL,
	"identifier" varchar(100) NOT NULL,
	"created_on" timestamp NOT NULL,
	"last_used_on" timestamp
).
2014/06/29 12:52:47 [error] [system.db.CDbCommand] Error in preparing SQL: CREATE TABLE '{{user_remote_identities}}' (
	"id" integer PRIMARY KEY AUTOINCREMENT NOT NULL,
	"user_id" integer NOT NULL REFERENCES {{users}} (id) ON UPDATE CASCADE ON DELETE CASCADE,
	"provider" varchar(100) NOT NULL,
	"identifier" varchar(100) NOT NULL,
	"created_on" timestamp NOT NULL,
	"last_used_on" timestamp
)
2014/06/29 12:52:47 [error] [system.db.CDbCommand] CDbCommand::execute() failed: CDbCommand failed to prepare the SQL statement: SQLSTATE[HY000]: General error: 1 unrecognized token: "{". The SQL statement executed was: CREATE TABLE '{{user_remote_identities}}' (
	"id" integer PRIMARY KEY AUTOINCREMENT NOT NULL,
	"user_id" integer NOT NULL REFERENCES {{users}} (id) ON UPDATE CASCADE ON DELETE CASCADE,
	"provider" varchar(100) NOT NULL,
	"identifier" varchar(100) NOT NULL,
	"created_on" timestamp NOT NULL,
	"last_used_on" timestamp
).
2014/06/29 12:53:15 [error] [system.db.CDbCommand] Error in preparing SQL: CREATE TABLE '{{user_remote_identities}}' (
	"id" integer PRIMARY KEY AUTOINCREMENT NOT NULL,
	"user_id" integer NOT NULL REFERENCES {{users}} (id) ON UPDATE CASCADE ON DELETE CASCADE,
	"provider" varchar(100) NOT NULL,
	"identifier" varchar(100) NOT NULL,
	"created_on" timestamp NOT NULL,
	"last_used_on" timestamp
)
2014/06/29 12:53:15 [error] [system.db.CDbCommand] CDbCommand::execute() failed: CDbCommand failed to prepare the SQL statement: SQLSTATE[HY000]: General error: 1 unrecognized token: "{". The SQL statement executed was: CREATE TABLE '{{user_remote_identities}}' (
	"id" integer PRIMARY KEY AUTOINCREMENT NOT NULL,
	"user_id" integer NOT NULL REFERENCES {{users}} (id) ON UPDATE CASCADE ON DELETE CASCADE,
	"provider" varchar(100) NOT NULL,
	"identifier" varchar(100) NOT NULL,
	"created_on" timestamp NOT NULL,
	"last_used_on" timestamp
).
2014/06/29 13:00:12 [error] [system.db.CDbCommand] Error in preparing SQL: CREATE TABLE '{user_remote_identities}' (
	"id" integer PRIMARY KEY AUTOINCREMENT NOT NULL,
	"user_id" integer NOT NULL REFERENCES {{users}} (id) ON UPDATE CASCADE ON DELETE CASCADE,
	"provider" varchar(100) NOT NULL,
	"identifier" varchar(100) NOT NULL,
	"created_on" timestamp NOT NULL,
	"last_used_on" timestamp
)
2014/06/29 13:00:12 [error] [system.db.CDbCommand] CDbCommand::execute() failed: CDbCommand failed to prepare the SQL statement: SQLSTATE[HY000]: General error: 1 unrecognized token: "{". The SQL statement executed was: CREATE TABLE '{user_remote_identities}' (
	"id" integer PRIMARY KEY AUTOINCREMENT NOT NULL,
	"user_id" integer NOT NULL REFERENCES {{users}} (id) ON UPDATE CASCADE ON DELETE CASCADE,
	"provider" varchar(100) NOT NULL,
	"identifier" varchar(100) NOT NULL,
	"created_on" timestamp NOT NULL,
	"last_used_on" timestamp
).
2014/06/29 13:00:58 [error] [system.db.CDbCommand] Error in preparing SQL: CREATE TABLE 'user_remote_identities' (
	"id" integer PRIMARY KEY AUTOINCREMENT NOT NULL,
	"user_id" integer NOT NULL REFERENCES {{users}} (id) ON UPDATE CASCADE ON DELETE CASCADE,
	"provider" varchar(100) NOT NULL,
	"identifier" varchar(100) NOT NULL,
	"created_on" timestamp NOT NULL,
	"last_used_on" timestamp
)
2014/06/29 13:00:58 [error] [system.db.CDbCommand] CDbCommand::execute() failed: CDbCommand failed to prepare the SQL statement: SQLSTATE[HY000]: General error: 1 unrecognized token: "{". The SQL statement executed was: CREATE TABLE 'user_remote_identities' (
	"id" integer PRIMARY KEY AUTOINCREMENT NOT NULL,
	"user_id" integer NOT NULL REFERENCES {{users}} (id) ON UPDATE CASCADE ON DELETE CASCADE,
	"provider" varchar(100) NOT NULL,
	"identifier" varchar(100) NOT NULL,
	"created_on" timestamp NOT NULL,
	"last_used_on" timestamp
).
2014/06/29 13:09:47 [error] [system.db.CDbCommand] Error in preparing SQL: CREATE TABLE '{{user_used_passwords}}' (
	"id" integer PRIMARY KEY AUTOINCREMENT NOT NULL,
	"user_id" integer NOT NULL REFERENCES {{users}} (id) ON UPDATE CASCADE ON DELETE CASCADE,
	"password" varchar(255) NOT NULL,
	"set_on" timestamp NOT NULL
)
2014/06/29 13:09:47 [error] [system.db.CDbCommand] CDbCommand::execute() failed: CDbCommand failed to prepare the SQL statement: SQLSTATE[HY000]: General error: 1 unrecognized token: "{". The SQL statement executed was: CREATE TABLE '{{user_used_passwords}}' (
	"id" integer PRIMARY KEY AUTOINCREMENT NOT NULL,
	"user_id" integer NOT NULL REFERENCES {{users}} (id) ON UPDATE CASCADE ON DELETE CASCADE,
	"password" varchar(255) NOT NULL,
	"set_on" timestamp NOT NULL
).
2014/06/29 13:17:29 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'purchaseorder.tbl_migration' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `tbl_migration`.
2014/06/29 13:25:24 [error] [exception.CException] exception 'CException' with message 'Alias "usr.UsrModule" is invalid. Make sure it points to an existing PHP file and the file is readable.' in C:\xampp\htdocs\yii\framework\YiiBase.php:322
Stack trace:
#0 C:\xampp\htdocs\yii\framework\YiiBase.php(196): YiiBase::import('usr.UsrModule', true)
#1 C:\xampp\htdocs\yii\framework\base\CModule.php(282): YiiBase::createComponent('usr.UsrModule', 'usr', NULL, Array)
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(337): CModule->getModule('usr')
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(276): CWebApplication->createController('usr')
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('usr')
#5 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#6 C:\xampp\htdocs\yii\framework\purchaseorder\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/yii/framework/purchaseorder/usr
---
2014/06/29 13:25:49 [error] [php] include(C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\UsrController.php): failed to open stream: No such file or directory (C:\xampp\htdocs\yii\framework\YiiBase.php:401)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\purchaseorder\protected\controllers\ManagerController.php(6): spl_autoload_call()
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(354): require()
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(276): CWebApplication->createController()
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#5 C:\xampp\htdocs\yii\framework\purchaseorder\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/framework/purchaseorder/manager
2014/06/29 13:29:27 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'purchaseorder.{{users}}' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `{{users}}`.
2014/06/29 13:29:27 [error] [exception.CDbException] exception 'CDbException' with message 'The table "{{users}}" for active record class "User" cannot be found in the database.' in C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php:2362
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(User))
#1 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(658): CActiveRecord->getMetaData()
#2 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1535): CActiveRecord->getTableSchema()
#3 C:\xampp\htdocs\yii\framework\purchaseorder\protected\components\UserIdentity.php(236): CActiveRecord->findAllByAttributes(Array)
#4 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\models\ProfileForm.php(113): UserIdentity::find(Array)
#5 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): ProfileForm->uniqueIdentity('username', Array)
#6 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(ProfileForm), 'username')
#7 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(ProfileForm), NULL)
#8 C:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(920): CModel->validate(NULL)
#9 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(260): CActiveForm::validate(Array)
#10 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DefaultController->actionRegister()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#13 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(34): CController->run('register')
#15 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): DefaultController->run('register')
#16 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('usr/default/reg...')
#17 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 C:\xampp\htdocs\yii\framework\purchaseorder\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/yii/framework/purchaseorder/usr/default/register
HTTP_REFERER=http://localhost/yii/framework/purchaseorder/usr/default/register
---
2014/06/29 13:29:29 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'purchaseorder.{{users}}' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `{{users}}`.
2014/06/29 13:29:29 [error] [exception.CDbException] exception 'CDbException' with message 'The table "{{users}}" for active record class "User" cannot be found in the database.' in C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php:2362
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(User))
#1 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(658): CActiveRecord->getMetaData()
#2 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1535): CActiveRecord->getTableSchema()
#3 C:\xampp\htdocs\yii\framework\purchaseorder\protected\components\UserIdentity.php(236): CActiveRecord->findAllByAttributes(Array)
#4 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\models\ProfileForm.php(113): UserIdentity::find(Array)
#5 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): ProfileForm->uniqueIdentity('username', Array)
#6 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(ProfileForm), 'username')
#7 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(ProfileForm), NULL)
#8 C:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(920): CModel->validate(NULL)
#9 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(260): CActiveForm::validate(Array)
#10 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DefaultController->actionRegister()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#13 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(34): CController->run('register')
#15 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): DefaultController->run('register')
#16 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('usr/default/reg...')
#17 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 C:\xampp\htdocs\yii\framework\purchaseorder\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/yii/framework/purchaseorder/usr/default/register
HTTP_REFERER=http://localhost/yii/framework/purchaseorder/usr/default/register
---
2014/06/29 13:29:36 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'purchaseorder.{{users}}' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `{{users}}`.
2014/06/29 13:29:36 [error] [exception.CDbException] exception 'CDbException' with message 'The table "{{users}}" for active record class "User" cannot be found in the database.' in C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php:2362
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(User))
#1 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(658): CActiveRecord->getMetaData()
#2 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1535): CActiveRecord->getTableSchema()
#3 C:\xampp\htdocs\yii\framework\purchaseorder\protected\components\UserIdentity.php(236): CActiveRecord->findAllByAttributes(Array)
#4 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\models\ProfileForm.php(113): UserIdentity::find(Array)
#5 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): ProfileForm->uniqueIdentity('username', Array)
#6 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(ProfileForm), 'username')
#7 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(ProfileForm), NULL)
#8 C:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(920): CModel->validate(NULL)
#9 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(260): CActiveForm::validate(Array)
#10 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DefaultController->actionRegister()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#13 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(34): CController->run('register')
#15 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): DefaultController->run('register')
#16 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('usr/default/reg...')
#17 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 C:\xampp\htdocs\yii\framework\purchaseorder\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/yii/framework/purchaseorder/usr/default/register
HTTP_REFERER=http://localhost/yii/framework/purchaseorder/usr/default/register
---
2014/06/29 13:29:39 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'purchaseorder.{{users}}' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `{{users}}`.
2014/06/29 13:29:39 [error] [exception.CDbException] exception 'CDbException' with message 'The table "{{users}}" for active record class "User" cannot be found in the database.' in C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php:2362
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(User))
#1 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(658): CActiveRecord->getMetaData()
#2 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1535): CActiveRecord->getTableSchema()
#3 C:\xampp\htdocs\yii\framework\purchaseorder\protected\components\UserIdentity.php(236): CActiveRecord->findAllByAttributes(Array)
#4 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\models\ProfileForm.php(113): UserIdentity::find(Array)
#5 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): ProfileForm->uniqueIdentity('username', Array)
#6 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(ProfileForm), 'username')
#7 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(ProfileForm), NULL)
#8 C:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(920): CModel->validate(NULL)
#9 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(260): CActiveForm::validate(Array)
#10 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DefaultController->actionRegister()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#13 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(34): CController->run('register')
#15 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): DefaultController->run('register')
#16 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('usr/default/reg...')
#17 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 C:\xampp\htdocs\yii\framework\purchaseorder\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/yii/framework/purchaseorder/usr/default/register
HTTP_REFERER=http://localhost/yii/framework/purchaseorder/usr/default/register
---
2014/06/29 13:29:43 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'purchaseorder.{{users}}' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `{{users}}`.
2014/06/29 13:29:43 [error] [exception.CDbException] exception 'CDbException' with message 'The table "{{users}}" for active record class "User" cannot be found in the database.' in C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php:2362
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(User))
#1 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(658): CActiveRecord->getMetaData()
#2 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1535): CActiveRecord->getTableSchema()
#3 C:\xampp\htdocs\yii\framework\purchaseorder\protected\components\UserIdentity.php(236): CActiveRecord->findAllByAttributes(Array)
#4 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\models\ProfileForm.php(113): UserIdentity::find(Array)
#5 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): ProfileForm->uniqueIdentity('username', Array)
#6 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(ProfileForm), 'username')
#7 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(ProfileForm), NULL)
#8 C:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(920): CModel->validate(NULL)
#9 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(260): CActiveForm::validate(Array)
#10 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DefaultController->actionRegister()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#13 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(34): CController->run('register')
#15 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): DefaultController->run('register')
#16 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('usr/default/reg...')
#17 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 C:\xampp\htdocs\yii\framework\purchaseorder\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/yii/framework/purchaseorder/usr/default/register
HTTP_REFERER=http://localhost/yii/framework/purchaseorder/usr/default/register
---
2014/06/29 13:29:45 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'purchaseorder.{{users}}' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `{{users}}`.
2014/06/29 13:29:45 [error] [exception.CDbException] exception 'CDbException' with message 'The table "{{users}}" for active record class "User" cannot be found in the database.' in C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php:2362
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(User))
#1 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(658): CActiveRecord->getMetaData()
#2 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1535): CActiveRecord->getTableSchema()
#3 C:\xampp\htdocs\yii\framework\purchaseorder\protected\components\UserIdentity.php(236): CActiveRecord->findAllByAttributes(Array)
#4 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\models\ProfileForm.php(113): UserIdentity::find(Array)
#5 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): ProfileForm->uniqueIdentity('username', Array)
#6 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(ProfileForm), 'username')
#7 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(ProfileForm), NULL)
#8 C:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(920): CModel->validate(NULL)
#9 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(260): CActiveForm::validate(Array)
#10 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DefaultController->actionRegister()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#13 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(34): CController->run('register')
#15 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): DefaultController->run('register')
#16 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('usr/default/reg...')
#17 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 C:\xampp\htdocs\yii\framework\purchaseorder\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/yii/framework/purchaseorder/usr/default/register
HTTP_REFERER=http://localhost/yii/framework/purchaseorder/usr/default/register
---
2014/06/29 13:29:59 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'purchaseorder.{{users}}' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `{{users}}`.
2014/06/29 13:29:59 [error] [exception.CDbException] exception 'CDbException' with message 'The table "{{users}}" for active record class "User" cannot be found in the database.' in C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php:2362
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(User))
#1 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(658): CActiveRecord->getMetaData()
#2 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1535): CActiveRecord->getTableSchema()
#3 C:\xampp\htdocs\yii\framework\purchaseorder\protected\components\UserIdentity.php(236): CActiveRecord->findAllByAttributes(Array)
#4 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\models\ProfileForm.php(113): UserIdentity::find(Array)
#5 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): ProfileForm->uniqueIdentity('username', Array)
#6 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(ProfileForm), 'username')
#7 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(ProfileForm), NULL)
#8 C:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(920): CModel->validate(NULL)
#9 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(260): CActiveForm::validate(Array)
#10 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DefaultController->actionRegister()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#13 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(34): CController->run('register')
#15 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): DefaultController->run('register')
#16 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('usr/default/reg...')
#17 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 C:\xampp\htdocs\yii\framework\purchaseorder\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/yii/framework/purchaseorder/usr/default/register
HTTP_REFERER=http://localhost/yii/framework/purchaseorder/usr/default/register
---
2014/06/29 13:30:07 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'purchaseorder.{{users}}' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `{{users}}`.
2014/06/29 13:30:07 [error] [exception.CDbException] exception 'CDbException' with message 'The table "{{users}}" for active record class "User" cannot be found in the database.' in C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php:2362
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(User))
#1 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(658): CActiveRecord->getMetaData()
#2 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1535): CActiveRecord->getTableSchema()
#3 C:\xampp\htdocs\yii\framework\purchaseorder\protected\components\UserIdentity.php(236): CActiveRecord->findAllByAttributes(Array)
#4 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\models\ProfileForm.php(113): UserIdentity::find(Array)
#5 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): ProfileForm->uniqueIdentity('username', Array)
#6 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(ProfileForm), 'username')
#7 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(ProfileForm), NULL)
#8 C:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(920): CModel->validate(NULL)
#9 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(260): CActiveForm::validate(Array)
#10 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DefaultController->actionRegister()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#13 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(34): CController->run('register')
#15 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): DefaultController->run('register')
#16 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('usr/default/reg...')
#17 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 C:\xampp\htdocs\yii\framework\purchaseorder\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/yii/framework/purchaseorder/usr/default/register
HTTP_REFERER=http://localhost/yii/framework/purchaseorder/usr/default/register
---
2014/06/29 13:31:10 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'purchaseorder.{{users}}' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `{{users}}`.
2014/06/29 13:31:10 [error] [exception.CDbException] exception 'CDbException' with message 'The table "{{users}}" for active record class "User" cannot be found in the database.' in C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php:2362
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(User))
#1 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(658): CActiveRecord->getMetaData()
#2 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1535): CActiveRecord->getTableSchema()
#3 C:\xampp\htdocs\yii\framework\purchaseorder\protected\components\UserIdentity.php(236): CActiveRecord->findAllByAttributes(Array)
#4 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\models\ProfileForm.php(113): UserIdentity::find(Array)
#5 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): ProfileForm->uniqueIdentity('username', Array)
#6 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(ProfileForm), 'username')
#7 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(ProfileForm), NULL)
#8 C:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(920): CModel->validate(NULL)
#9 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(260): CActiveForm::validate(Array)
#10 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DefaultController->actionRegister()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#13 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(34): CController->run('register')
#15 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): DefaultController->run('register')
#16 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('usr/default/reg...')
#17 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 C:\xampp\htdocs\yii\framework\purchaseorder\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/yii/framework/purchaseorder/usr/default/register
HTTP_REFERER=http://localhost/yii/framework/purchaseorder/usr/default/register
---
2014/06/29 13:33:31 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'purchaseorder.{{users}}' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `{{users}}`.
2014/06/29 13:33:31 [error] [exception.CDbException] exception 'CDbException' with message 'The table "{{users}}" for active record class "User" cannot be found in the database.' in C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php:2362
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(User))
#1 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(658): CActiveRecord->getMetaData()
#2 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1535): CActiveRecord->getTableSchema()
#3 C:\xampp\htdocs\yii\framework\purchaseorder\protected\components\UserIdentity.php(236): CActiveRecord->findAllByAttributes(Array)
#4 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\models\ProfileForm.php(113): UserIdentity::find(Array)
#5 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): ProfileForm->uniqueIdentity('username', Array)
#6 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(ProfileForm), 'username')
#7 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(ProfileForm), NULL)
#8 C:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(920): CModel->validate(NULL)
#9 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(260): CActiveForm::validate(Array)
#10 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DefaultController->actionRegister()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#13 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(34): CController->run('register')
#15 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): DefaultController->run('register')
#16 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('usr/default/reg...')
#17 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 C:\xampp\htdocs\yii\framework\purchaseorder\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/yii/framework/purchaseorder/usr/default/register
HTTP_REFERER=http://localhost/yii/framework/purchaseorder/usr/default/register
---
2014/06/29 13:37:07 [error] [php] include(UserIdentity.php): failed to open stream: No such file or directory (C:\xampp\htdocs\yii\framework\YiiBase.php:427)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\models\ProfileForm.php(113): spl_autoload_call()
#1 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): ProfileForm->uniqueIdentity()
#2 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute()
#3 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CInlineValidator->validate()
#4 C:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(920): ProfileForm->validate()
#5 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(260): validate()
#6 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DefaultController->actionRegister()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(286): DefaultController->runAction()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(265): DefaultController->runActionWithFilters()
#10 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(34): DefaultController->run()
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): DefaultController->run()
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#13 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#14 C:\xampp\htdocs\yii\framework\purchaseorder\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/framework/purchaseorder/usr/default/register
2014/06/29 13:37:11 [error] [php] include(UserIdentity.php): failed to open stream: No such file or directory (C:\xampp\htdocs\yii\framework\YiiBase.php:427)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\models\ProfileForm.php(91): spl_autoload_call()
#1 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\views\default\_form.php(44): ProfileForm->getIdentity()
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require()
#3 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): DefaultController->renderInternal()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(869): DefaultController->renderFile()
#5 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\views\default\updateProfile.php(32): DefaultController->renderPartial()
#6 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require()
#7 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): DefaultController->renderInternal()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(869): DefaultController->renderFile()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(782): DefaultController->renderPartial()
#10 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(298): DefaultController->render()
#11 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DefaultController->actionRegister()
#12 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#13 C:\xampp\htdocs\yii\framework\web\CController.php(286): DefaultController->runAction()
#14 C:\xampp\htdocs\yii\framework\web\CController.php(265): DefaultController->runActionWithFilters()
#15 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(34): DefaultController->run()
#16 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): DefaultController->run()
#17 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#18 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#19 C:\xampp\htdocs\yii\framework\purchaseorder\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/framework/purchaseorder/usr/default/register
2014/06/29 13:38:10 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'purchaseorder.{{users}}' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `{{users}}`.
2014/06/29 13:38:10 [error] [exception.CDbException] exception 'CDbException' with message 'The table "{{users}}" for active record class "User" cannot be found in the database.' in C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php:2362
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(User))
#1 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(658): CActiveRecord->getMetaData()
#2 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1535): CActiveRecord->getTableSchema()
#3 C:\xampp\htdocs\yii\framework\purchaseorder\protected\components\UserIdentity.php(236): CActiveRecord->findAllByAttributes(Array)
#4 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\models\ProfileForm.php(113): UserIdentity::find(Array)
#5 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): ProfileForm->uniqueIdentity('username', Array)
#6 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(ProfileForm), 'username')
#7 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(ProfileForm), NULL)
#8 C:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(920): CModel->validate(NULL)
#9 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(260): CActiveForm::validate(Array)
#10 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DefaultController->actionRegister()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#13 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(34): CController->run('register')
#15 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): DefaultController->run('register')
#16 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('usr/default/reg...')
#17 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 C:\xampp\htdocs\yii\framework\purchaseorder\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/yii/framework/purchaseorder/usr/default/register
HTTP_REFERER=http://localhost/yii/framework/purchaseorder/usr/default/register
---
2014/06/29 13:38:12 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'purchaseorder.{{users}}' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `{{users}}`.
2014/06/29 13:38:12 [error] [exception.CDbException] exception 'CDbException' with message 'The table "{{users}}" for active record class "User" cannot be found in the database.' in C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php:2362
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(User))
#1 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(658): CActiveRecord->getMetaData()
#2 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1535): CActiveRecord->getTableSchema()
#3 C:\xampp\htdocs\yii\framework\purchaseorder\protected\components\UserIdentity.php(236): CActiveRecord->findAllByAttributes(Array)
#4 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\models\ProfileForm.php(113): UserIdentity::find(Array)
#5 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): ProfileForm->uniqueIdentity('username', Array)
#6 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(ProfileForm), 'username')
#7 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(ProfileForm), NULL)
#8 C:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(920): CModel->validate(NULL)
#9 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(260): CActiveForm::validate(Array)
#10 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DefaultController->actionRegister()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#13 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(34): CController->run('register')
#15 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): DefaultController->run('register')
#16 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('usr/default/reg...')
#17 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 C:\xampp\htdocs\yii\framework\purchaseorder\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/yii/framework/purchaseorder/usr/default/register
HTTP_REFERER=http://localhost/yii/framework/purchaseorder/usr/default/register
---
2014/06/29 13:38:19 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'purchaseorder.{{users}}' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `{{users}}`.
2014/06/29 13:38:19 [error] [exception.CDbException] exception 'CDbException' with message 'The table "{{users}}" for active record class "User" cannot be found in the database.' in C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php:2362
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(User))
#1 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(658): CActiveRecord->getMetaData()
#2 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1535): CActiveRecord->getTableSchema()
#3 C:\xampp\htdocs\yii\framework\purchaseorder\protected\components\UserIdentity.php(236): CActiveRecord->findAllByAttributes(Array)
#4 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\models\ProfileForm.php(113): UserIdentity::find(Array)
#5 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): ProfileForm->uniqueIdentity('username', Array)
#6 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(ProfileForm), 'username')
#7 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(ProfileForm), NULL)
#8 C:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(920): CModel->validate(NULL)
#9 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(260): CActiveForm::validate(Array)
#10 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DefaultController->actionRegister()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#13 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(34): CController->run('register')
#15 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): DefaultController->run('register')
#16 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('usr/default/reg...')
#17 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 C:\xampp\htdocs\yii\framework\purchaseorder\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/yii/framework/purchaseorder/usr/default/register
HTTP_REFERER=http://localhost/yii/framework/purchaseorder/usr/default/register
---
2014/06/29 13:38:22 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'purchaseorder.{{users}}' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `{{users}}`.
2014/06/29 13:38:22 [error] [exception.CDbException] exception 'CDbException' with message 'The table "{{users}}" for active record class "User" cannot be found in the database.' in C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php:2362
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(User))
#1 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(658): CActiveRecord->getMetaData()
#2 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1535): CActiveRecord->getTableSchema()
#3 C:\xampp\htdocs\yii\framework\purchaseorder\protected\components\UserIdentity.php(236): CActiveRecord->findAllByAttributes(Array)
#4 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\models\ProfileForm.php(113): UserIdentity::find(Array)
#5 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): ProfileForm->uniqueIdentity('username', Array)
#6 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(ProfileForm), 'username')
#7 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(ProfileForm), NULL)
#8 C:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(920): CModel->validate(NULL)
#9 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(260): CActiveForm::validate(Array)
#10 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DefaultController->actionRegister()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#13 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(34): CController->run('register')
#15 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): DefaultController->run('register')
#16 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('usr/default/reg...')
#17 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 C:\xampp\htdocs\yii\framework\purchaseorder\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/yii/framework/purchaseorder/usr/default/register
HTTP_REFERER=http://localhost/yii/framework/purchaseorder/usr/default/register
---
2014/06/29 13:38:23 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'purchaseorder.{{users}}' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `{{users}}`.
2014/06/29 13:38:23 [error] [exception.CDbException] exception 'CDbException' with message 'The table "{{users}}" for active record class "User" cannot be found in the database.' in C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php:2362
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(User))
#1 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(658): CActiveRecord->getMetaData()
#2 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1535): CActiveRecord->getTableSchema()
#3 C:\xampp\htdocs\yii\framework\purchaseorder\protected\components\UserIdentity.php(236): CActiveRecord->findAllByAttributes(Array)
#4 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\models\ProfileForm.php(113): UserIdentity::find(Array)
#5 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): ProfileForm->uniqueIdentity('username', Array)
#6 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(ProfileForm), 'username')
#7 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(ProfileForm), NULL)
#8 C:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(920): CModel->validate(NULL)
#9 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(260): CActiveForm::validate(Array)
#10 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DefaultController->actionRegister()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#13 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(34): CController->run('register')
#15 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): DefaultController->run('register')
#16 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('usr/default/reg...')
#17 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 C:\xampp\htdocs\yii\framework\purchaseorder\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/yii/framework/purchaseorder/usr/default/register
HTTP_REFERER=http://localhost/yii/framework/purchaseorder/usr/default/register
---
2014/06/29 13:38:24 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'purchaseorder.{{users}}' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `{{users}}`.
2014/06/29 13:38:24 [error] [exception.CDbException] exception 'CDbException' with message 'The table "{{users}}" for active record class "User" cannot be found in the database.' in C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php:2362
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(User))
#1 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(658): CActiveRecord->getMetaData()
#2 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1535): CActiveRecord->getTableSchema()
#3 C:\xampp\htdocs\yii\framework\purchaseorder\protected\components\UserIdentity.php(236): CActiveRecord->findAllByAttributes(Array)
#4 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\models\ProfileForm.php(113): UserIdentity::find(Array)
#5 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): ProfileForm->uniqueIdentity('username', Array)
#6 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(ProfileForm), 'username')
#7 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(ProfileForm), NULL)
#8 C:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(920): CModel->validate(NULL)
#9 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(260): CActiveForm::validate(Array)
#10 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DefaultController->actionRegister()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#13 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(34): CController->run('register')
#15 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): DefaultController->run('register')
#16 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('usr/default/reg...')
#17 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 C:\xampp\htdocs\yii\framework\purchaseorder\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/yii/framework/purchaseorder/usr/default/register
HTTP_REFERER=http://localhost/yii/framework/purchaseorder/usr/default/register
---
2014/06/29 13:38:26 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'purchaseorder.{{users}}' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `{{users}}`.
2014/06/29 13:38:26 [error] [exception.CDbException] exception 'CDbException' with message 'The table "{{users}}" for active record class "User" cannot be found in the database.' in C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php:2362
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(User))
#1 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(658): CActiveRecord->getMetaData()
#2 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1535): CActiveRecord->getTableSchema()
#3 C:\xampp\htdocs\yii\framework\purchaseorder\protected\components\UserIdentity.php(236): CActiveRecord->findAllByAttributes(Array)
#4 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\models\ProfileForm.php(113): UserIdentity::find(Array)
#5 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): ProfileForm->uniqueIdentity('username', Array)
#6 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(ProfileForm), 'username')
#7 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(ProfileForm), NULL)
#8 C:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(920): CModel->validate(NULL)
#9 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(260): CActiveForm::validate(Array)
#10 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DefaultController->actionRegister()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#13 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(34): CController->run('register')
#15 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): DefaultController->run('register')
#16 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('usr/default/reg...')
#17 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 C:\xampp\htdocs\yii\framework\purchaseorder\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/yii/framework/purchaseorder/usr/default/register
HTTP_REFERER=http://localhost/yii/framework/purchaseorder/usr/default/register
---
2014/06/29 13:40:28 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'purchaseorder.{{users}}' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `{{users}}`.
2014/06/29 13:40:28 [error] [exception.CDbException] exception 'CDbException' with message 'The table "{{users}}" for active record class "User" cannot be found in the database.' in C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php:2362
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(User))
#1 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(658): CActiveRecord->getMetaData()
#2 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1535): CActiveRecord->getTableSchema()
#3 C:\xampp\htdocs\yii\framework\purchaseorder\protected\components\UserIdentity.php(236): CActiveRecord->findAllByAttributes(Array)
#4 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\models\ProfileForm.php(113): UserIdentity::find(Array)
#5 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): ProfileForm->uniqueIdentity('username', Array)
#6 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(ProfileForm), 'username')
#7 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(ProfileForm), NULL)
#8 C:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(920): CModel->validate(NULL)
#9 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(260): CActiveForm::validate(Array)
#10 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DefaultController->actionRegister()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#13 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(34): CController->run('register')
#15 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): DefaultController->run('register')
#16 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('usr/default/reg...')
#17 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 C:\xampp\htdocs\yii\framework\purchaseorder\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/yii/framework/purchaseorder/usr/default/register
HTTP_REFERER=http://localhost/yii/framework/purchaseorder/usr/default/register
---
2014/06/29 13:40:30 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'purchaseorder.{{users}}' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `{{users}}`.
2014/06/29 13:40:30 [error] [exception.CDbException] exception 'CDbException' with message 'The table "{{users}}" for active record class "User" cannot be found in the database.' in C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php:2362
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(User))
#1 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(658): CActiveRecord->getMetaData()
#2 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1535): CActiveRecord->getTableSchema()
#3 C:\xampp\htdocs\yii\framework\purchaseorder\protected\components\UserIdentity.php(236): CActiveRecord->findAllByAttributes(Array)
#4 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\models\ProfileForm.php(113): UserIdentity::find(Array)
#5 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): ProfileForm->uniqueIdentity('username', Array)
#6 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(ProfileForm), 'username')
#7 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(ProfileForm), NULL)
#8 C:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(920): CModel->validate(NULL)
#9 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(260): CActiveForm::validate(Array)
#10 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DefaultController->actionRegister()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#13 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\DefaultController.php(34): CController->run('register')
#15 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): DefaultController->run('register')
#16 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('usr/default/reg...')
#17 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 C:\xampp\htdocs\yii\framework\purchaseorder\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/yii/framework/purchaseorder/usr/default/register
HTTP_REFERER=http://localhost/yii/framework/purchaseorder/usr/default/register
---
2014/06/29 13:50:27 [error] [php] Trying to get property of non-object (C:\xampp\htdocs\yii\framework\purchaseorder\protected\controllers\SiteController.php:33)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#5 C:\xampp\htdocs\yii\framework\purchaseorder\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/framework/purchaseorder/
2014/06/29 13:52:20 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('manager')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii\framework\purchaseorder\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii/framework/purchaseorder/manager
---
2014/06/29 13:58:40 [error] [exception.Exception] exception 'Exception' with message 'Your application id and secret are required in order to connect to Facebook.' in C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\extensions\Hybrid\Providers\Facebook.php:26
Stack trace:
#0 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\extensions\Hybrid\Provider_Model.php(69): Hybrid_Providers_Facebook->initialize()
#1 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\extensions\Hybrid\Provider_Adapter.php(85): Hybrid_Provider_Model->__construct('Facebook', Array, Array)
#2 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\extensions\Hybrid\Auth.php(270): Hybrid_Provider_Adapter->factory('facebook', Array)
#3 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\extensions\Hybrid\Auth.php(215): Hybrid_Auth::setup('facebook', Array)
#4 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\models\HybridauthForm.php(114): Hybrid_Auth::authenticate('facebook', Array)
#5 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\HybridauthController.php(40): HybridauthForm->login()
#6 [internal function]: HybridauthController->actionPopup('Facebook')
#7 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(HybridauthController), Array)
#8 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(HybridauthController), Object(ReflectionMethod), Array)
#9 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('popup')
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('usr/hybridauth/...')
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\yii\framework\purchaseorder\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yii/framework/purchaseorder/usr/hybridauth/popup/provider/Facebook
HTTP_REFERER=http://localhost/yii/framework/purchaseorder/usr/default/login
---
2014/06/29 13:59:03 [error] [exception.Exception] exception 'Exception' with message 'Your application id and secret are required in order to connect to Facebook.' in C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\extensions\Hybrid\Providers\Facebook.php:26
Stack trace:
#0 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\extensions\Hybrid\Provider_Model.php(69): Hybrid_Providers_Facebook->initialize()
#1 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\extensions\Hybrid\Provider_Adapter.php(85): Hybrid_Provider_Model->__construct('Facebook', Array, Array)
#2 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\extensions\Hybrid\Auth.php(270): Hybrid_Provider_Adapter->factory('facebook', Array)
#3 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\extensions\Hybrid\Auth.php(215): Hybrid_Auth::setup('facebook', Array)
#4 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\models\HybridauthForm.php(114): Hybrid_Auth::authenticate('facebook', Array)
#5 C:\xampp\htdocs\yii\framework\purchaseorder\protected\modules\usr\controllers\HybridauthController.php(40): HybridauthForm->login()
#6 [internal function]: HybridauthController->actionPopup('Facebook')
#7 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(HybridauthController), Array)
#8 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(HybridauthController), Object(ReflectionMethod), Array)
#9 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('popup')
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('usr/hybridauth/...')
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\yii\framework\purchaseorder\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yii/framework/purchaseorder/usr/hybridauth/popup/provider/Facebook
HTTP_REFERER=http://localhost/yii/framework/purchaseorder/usr/default/login
---
2014/06/29 16:02:59 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('manager')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii\framework\purchaseorder\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii/framework/purchaseorder/manager
HTTP_REFERER=http://localhost/yii/framework/purchaseorder/usr/default/login
---
2014/06/29 18:17:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "inquiry".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('inquiry')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\framework\purchaseorder\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/framework/purchaseorder/inquiry
---
2014/06/29 18:27:27 [error] [exception.CException] exception 'CException' with message 'Property "CTimestampBehavior.modifyAttribute" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\yii\framework\YiiBase.php(220): CComponent->__set('modifyAttribute', 'modified_date')
#1 C:\xampp\htdocs\yii\framework\base\CComponent.php(329): YiiBase::createComponent(Array)
#2 C:\xampp\htdocs\yii\framework\base\CComponent.php(298): CComponent->attachBehavior('timestamps', Array)
#3 C:\xampp\htdocs\yii\framework\web\CController.php(114): CComponent->attachBehaviors(Array)
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(359): CController->__construct('inquiries', NULL)
#5 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(276): CWebApplication->createController('inquiries/creat...')
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('inquiries/creat...')
#7 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#8 C:\xampp\htdocs\yii\framework\purchaseorder\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/yii/framework/purchaseorder/inquiries/create
HTTP_REFERER=http://localhost/yii/framework/purchaseorder/inquiries/create
---
2014/06/29 18:27:48 [error] [exception.CException] exception 'CException' with message 'Event "InquiriesController.onBeforeSave" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:478
Stack trace:
#0 C:\xampp\htdocs\yii\framework\base\CComponent.php(516): CComponent->getEventHandlers('onBeforeSave')
#1 C:\xampp\htdocs\yii\framework\base\CBehavior.php(111): CComponent->attachEventHandler('onBeforeSave', Array)
#2 C:\xampp\htdocs\yii\framework\base\CBehavior.php(51): CBehavior->_attachEventHandlers()
#3 C:\xampp\htdocs\yii\framework\base\CComponent.php(331): CBehavior->attach(Object(InquiriesController))
#4 C:\xampp\htdocs\yii\framework\base\CComponent.php(298): CComponent->attachBehavior('timestamps', Array)
#5 C:\xampp\htdocs\yii\framework\web\CController.php(114): CComponent->attachBehaviors(Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(359): CController->__construct('inquiries', NULL)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(276): CWebApplication->createController('inquiries/creat...')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('inquiries/creat...')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii\framework\purchaseorder\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii/framework/purchaseorder/inquiries/create
HTTP_REFERER=http://localhost/yii/framework/purchaseorder/inquiries/create
---
2014/06/29 18:48:41 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('inquiries/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii\framework\purchaseorder\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii/framework/purchaseorder/inquiries/admin
HTTP_REFERER=http://localhost/yii/framework/purchaseorder/inquiries/index
---
